contracts/modules/legacy/LegacyPooledStaking.sol

```diff
@@ -1422,20 +1422,20 @@ contract LegacyPooledStaking is IPooledStaking, MasterAwareV2 {
     nxm.transfer(stakerAddress, nxmToBeUnlocked);
   }
 
-
   function migrateToPool(
     StakingPoolMigrationData memory migrationData,
     ProductInitializationParams[] memory params
   ) internal {
-    ( /* uint stakingPoolId */, address stakingPoolAddress) = cover.createStakingPool(
-      migrationData.stakerAddress,
+    (uint stakingPoolId, address stakingPoolAddress) = cover.createStakingPool(
       migrationData.isPrivatePool,
       migrationData.initialPoolFee,
       migrationData.maxPoolFee,
       params,
       migrationData.ipfsDescriptionHash
     );
 
+    tokenController().assignStakingPoolManager(stakingPoolId, migrationData.stakerAddress);
```

Comment:
migrationData has a parameter called migrationData.managerAddress

The fact that we don't use it here properly is a big that is fixed by the PR currently in progress.


```tokenController().assignStakingPoolManager(stakingPoolId, migrationData.managerAddress);```

Reply:
Was not aware that `stakerAddress` and `managerAddress` could be different so I just removed one of them. Do you want me to fix this here or you want to fix it in your PR?

Reply:
it's a simple fix of just replacing it with managerAddress, i'd do it here as well 

code will get merged after should be the same val

Reply:
Done